components/jaeger-collector.libsonnet

```diff
@@ -82,13 +82,13 @@ local jaegerAgent = import './jaeger-agent.libsonnet';
       local c =
         container.new($.jaeger.deployment.metadata.name, j.image) +
         container.withArgs([
-          '--badger.directory-key=/var/jaeger/store/keys',
-          '--badger.directory-value=/var/jaeger/store/values',
-          '--badger.ephemeral=false',
+          // '--badger.directory-key=/var/jaeger/store/keys',
```

Comment:
I think we have git history for this, but I know it's temporary. If anything, `TODO` would be nice (: